odoo/orm/models.py

```diff
@@ -1536,37 +1540,45 @@ def avoid(field):
         return defaults
 
     @api.model
-    def _read_group(self, domain: DomainType, groupby=(), aggregates=(), having=(), offset=0, limit=None, order=None) -> list[tuple]:
+    def _read_group(
+        self,
+        domain: DomainType,
+        groupby: Sequence[str] = (),
+        aggregates: Sequence[str] = (),
+        having: Sequence[str] = (),
```

Comment:
`having` is a domain on field aggregates.  A list of strings is not a valid value for `having`.